docs/news.rst

```diff
@@ -1,6 +1,14 @@
 Release Notes
 =============
 
+
+TBD
```

Comment:
We can assume it will be 1.14.0 and I typically just set the date as "(unreleased)".